templates/patterns/text_featured_media/pattern-text-featured-media.html.twig

```diff
@@ -16,7 +16,17 @@
 {% endif %}
 {% set text = text|render %}
 
-{% set media_position_class = 'right' in variant ? '' : ' ecl-u-order-m-last' %}
+{% if 'right' in variant %}
+  {% if highlighted == true %}
+    {% set media_class = 'ecl-u-pr-l ecl-u-pl-none' %}
+  {% else %}
+    {% set media_class = 'ecl-u-pl-none' %}
+  {% endif %}
+{% elseif highlighted == true %}
+  {% set media_class = 'ecl-u-order-m-last ecl-u-pr-none' %}
+{% else %}
+  {% set media_class = 'ecl-u-order-m-last' %}
```

Comment:
```suggestion
  {% set media_class = 'ecl-u-order-m-last ecl-u-pr-none' %}
```
It is just a proposal. With this class image is aligned with next block.
<img width="1261" alt="Screenshot 2022-04-06 at 19 00 54" src="https://user-images.githubusercontent.com/1183951/162017902-8f73bbc6-a68c-45ec-91d9-3cb6db2341e1.png">

Reply:
Nice, I fixed.